src/polymer/expression-scanner.ts

```diff
@@ -89,52 +105,59 @@ export class ScannedAttributeExpression implements ScannedFeature {
  * Find and parse Polymer databinding expressions in HTML.
  *
  * TODO(rictic):
- *  - parse expressions inline in strings rather than entire attributes
- *  - parse expressions in text nodes
+ *  - more precise source ranges for databinding expressions
  *  - allow getting source ranges inside of a databinding expression.
  *  - surface parse errors in expressions as warnings.
  */
 export class ExpressionScanner implements HtmlScanner {
   async scan(
       document: ParsedHtmlDocument, _: (visitor: HtmlVisitor) => Promise<void>):
-      Promise<ScannedAttributeExpression[]> {
-    const results: ScannedAttributeExpression[] = [];
+      Promise<ScannedDatabindingExpression[]> {
+    const results: ScannedDatabindingExpression[] = [];
     const dataBindingTemplates = getAllDataBindingTemplates(document.ast);
     for (const template of dataBindingTemplates) {
       dom5.nodeWalkAll(template.content, (node) => {
+        if (dom5.isTextNode(node)) {
+          const dataBindings = findDatabindingInString(node.value || '');
```

Comment:
switch on `node.value` outside of the call? no point in calling the function on an empty string